trunk/SUAVE/Analyses/Atmospheric/US_Standard_1976.py

```diff
@@ -0,0 +1,172 @@
+""" US_Standard_1976.py: U.S. Standard Atmosphere (1976) """
+#
+#
+# Modified by Tim MacDonald 2/16/15
+# Converted to vector form and changed output structure
+
+# ----------------------------------------------------------------------
+#  Imports
+# ----------------------------------------------------------------------
+
+import numpy as np
+from SUAVE.Attributes.Gases import Air
+from SUAVE.Attributes.Atmospheres import Atmosphere
+from SUAVE.Analyses.Atmospheric import Atmospheric
+from SUAVE.Attributes.Planets import Earth
+from SUAVE.Core import Data
+from SUAVE.Core import Units
+import SUAVE
+
+# ----------------------------------------------------------------------
+#  Classes
+# ----------------------------------------------------------------------
+
+class US_Standard_1976(Atmospheric):
+
+    """ Implements the U.S. Standard Atmosphere (1976 version)
+    """
+    
+    def __defaults__(self):
+        atmo_data = SUAVE.Attributes.Atmospheres.Earth.US_Standard_1976()
+        self.update(atmo_data)        
+    
+    def compute_values(self,altitude,type="all"):
+
+        """ Computes values from the International Standard Atmosphere
+
+        Inputs:
+            altitude     : geometric altitude (elevation) (m)
+                           can be a float, list or 1D array of floats
+         
+        Outputs:
+            list of conditions -
+                pressure       : static pressure (Pa)
+                temperature    : static temperature (K)
+                density        : density (kg/m^3)
+                speed_of_sound : speed of sound (m/s)
+                viscosity      : viscosity (kg/m-s)
+            
+        Example:
+            atmosphere = SUAVE.Attributes.Atmospheres.Earth.USStandard1976()
+            atmosphere.ComputeValues(1000).pressure
+          
+        """
+
+        # unpack
+        zs   = altitude
+        gas_base  = self.fluid_properties
+        grav_base = self.planet.sea_level_gravity
+        Rad_base  = self.planet.mean_radius
+        self.fluid_properties = Air()
+        self.planet = Earth()     
+        gas  = self.fluid_properties
+        grav = self.planet.sea_level_gravity        
+        Rad  = self.planet.mean_radius
+        if (gas_base != self.fluid_properties) or (grav_base != self.planet.sea_level_gravity) or (Rad_base != Rad):
+            print 'Warning: US Standard Atmosphere being used outside expected conditions'
+
+        # convert input if necessary
+        if isinstance(zs, int): 
```

Comment:
It seems that the array functions don't operate with 2d columns, but if I make this change and leave the one below as is it will work.